bundles/apidocs/pom.xml

```diff
@@ -294,12 +294,14 @@
                             <sourceFileExcludes>
                                 <fileExclude>META-INF/versions/21/org/glassfish/jersey/helidon/connector/*.java</fileExclude>
                                 <fileExclude>org/glassfish/jersey/helidon/connector/*.java</fileExclude>
+                                <fileExclude>org/glassfish/jersey/jetty/*.java</fileExclude>
```

Comment:
Is this correct? Jetty is no longer multirelease here

Reply:
fixed. Excludes which were used due to multi-release jars from previous releases are removed.